blueocean/src/test/java/io/jenkins/blueocean/config/JenkinsJSExtensionsTest.java

```diff
@@ -55,7 +55,7 @@ public void test() {
             Assert.assertNotNull(extensionPoints);
 
             if ("blueocean-dashboard".equals(pluginId)) {
-                Assert.assertEquals(7, extensionPoints.size());
+                Assert.assertEquals(9, extensionPoints.size());
```

Comment:
yeah - it is ancient as the rocks... I think it is fine to delete it if it helps. I probably add it so accept full responsibility.

*stabs self with sword*